s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
s3am can read credentials from the metadata service, via IAM roles. CGCloud supports that and Justin confirmed that it works. The question is whether the metadata service is reachable from within a container. As a fallback, credentials can be passed via environment variables. The Toil scripts for the SGDD team make heavy use of that.